src/main/java/gui/styling/StyledButton.java

```diff
@@ -6,19 +6,18 @@
 import javafx.beans.property.SimpleObjectProperty;
 import javafx.beans.property.SimpleStringProperty;
 import javafx.beans.property.StringProperty;
-import javafx.geometry.Point3D;
 import javafx.scene.control.Button;
 import javafx.scene.effect.BlurType;
 import javafx.scene.effect.DropShadow;
 import javafx.scene.paint.Color;
-import lombok.Getter;
-import lombok.Setter;
 
 /**
  * Represents a stylized button.
  * @author Mark
```

Comment:
Thought these were gone, think I added one by accident. Can you remove that @author annotation?